pyaedt/application/Design.py

```diff
@@ -3916,3 +3916,32 @@ def set_temporary_directory(self, temp_dir_path):
         """
         self.odesktop.SetTempDirectory(temp_dir_path)
         return True
+
+    @pyaedt_function_handler()
+    def design_settings(self):
+        """Get design settings for the current AEDT app.
+
+        Returns
+        -------
+        dict
+            Dictionary of valid design settings.
+
+        References
+        ----------
+
+        >>> oDesign.GetChildObject("Design Settings")
+        """
+        try:
+            design_settings = self._odesign.GetChildObject("Design Settings")
+        except Exception:
+            raise Exception("Could not use GetChildObject('Design Settings')")
+
+        prop_name_list = design_settings.GetPropNames()
+        design_settings_dict = {}
+        for prop in prop_name_list:
+            try:
+                design_settings_dict[prop] = design_settings.GetPropValue(prop)
+            except Exception:
+                design_settings_dict[prop] = "Could not retrieve the value."
```

Comment:
@nnetake same as above, return False

Reply:
In this case, I don't want to return False because it may happen that out of 20 it is not able to retrieve a few properties. This is a partial success and I have added a warning message here.